orc8r/gateway/python/setup.py

```diff
@@ -34,92 +34,6 @@ def os_release():
 # version this represents. 'None' defaults to 0.0.0.
 VERSION = os.environ.get('PKG_VERSION', None)
 
-release_info = os_release()
```

Comment:
should we remove the os_release function as well?

Reply:
I left it because it gives you the version as well. But if we want version-specific stuff later, we can bring it back I guess.

Reply:
Removed.